manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE
+PackageVersion: 0.0.6.178
+PackageLocale: en-US
+Publisher: Tencent Games

```

```suggestion
Publisher: Level Infinite
```
This should match control panel